### DISCUSSION TITLE
Tailwind and Next.js

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [tailwindlabs/tailwindcss#1859] [Help]
Hello everybody,
i'm trying to use new Tailwind's purge feature with Next.js, but since html doesn't have 'Class' attribute it's not working.
Do you know how can i do the trick?

**Comment by estevanmaito** [comment]
Did you set the right path to `purge`?

This is from a project of mine (using Next): 

```js
content: ['./pages/**/*.jsx', './pages/**/*.mdx', './containers/**/*.jsx', './components/**/*.jsx']
```

If you want to see more, here is the complete `tailwind.config.js`: https://github.com/estevanmaito/tailwind-starter-kit/blob/master/tailwind.config.js

**Reply by federicocappellotto97** [reply]
Thank you so much man! It's working!
